torch/csrc/cuda/shared/nvtx.cpp

```diff
@@ -6,6 +6,12 @@
 
 namespace torch::cuda::shared {
 
+void registerString(const char* string) {
+#ifndef USE_ROCM
+  nvtxDomainRegisterString(nullptr, string);
```

Comment:
isn't this supposed to return a handle that you can pass to things instead of a string?